### PR TITLE
Functional tests - Add new test 'Sign in in checkout page'

### PR DESCRIPTION
**Pull request by nesrineabdmouleh**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | 8.1.x
| Description?      | Add new test 'Sign in in checkout page'
| Type?             | refacto
| Category?         | TE
| BC breaks?        | no
| Deprecations?     | no
| How to test?      | no need
| Fixed ticket?     | no
| Related PRs       | no
| Sponsor company   |

### CODE REVIEW

File: tests/UI/pages/FO/checkout/index.ts [hunk 1]
```diff
@@ -328,6 +340,36 @@ class Checkout extends FOBasePage {
     await page.click(this.signInLink);
   }
 
+  /**
+   * Logout customer
+   * @param page {Page} Browser tab
```
Comment (Progi1984) [inline tests/UI/pages/FO/checkout/index.ts:354]:
```suggestion
   * @param page {Page} Browser tab
   * @returns Promise<boolean>
```

File: tests/UI/pages/FO/checkout/index.ts [hunk 2]
```diff
@@ -328,6 +340,36 @@ class Checkout extends FOBasePage {
...
+   */
+  async logOutCustomer(page: Page): Promise<boolean> {
+    await this.waitForSelectorAndClick(page, this.personalInformationLogoutLink);
+
+    return this.isStepCompleted(page, this.personalInformationStepForm);
+  }
+
+  /**
+   * Click on edit personal information step
+   * @param page {Page} Browser tab
```
Comment (Progi1984) [inline tests/UI/pages/FO/checkout/index.ts:365]:
```suggestion
   * @param page {Page} Browser tab
   * @returns Promise<void>
```

File: tests/UI/pages/FO/checkout/index.ts [hunk 3]
```diff
@@ -328,6 +340,36 @@ class Checkout extends FOBasePage {
...
+   */
+  async clickOnEditPersonalInformationStep(page: Page): Promise<void> {
+    await this.waitForSelectorAndClick(page, this.personalInformationEditLink);
+  }
+
+  /**
+   * Get customer identity
+   * @param page {Page} Browser tab
```
Comment (Progi1984) [inline tests/UI/pages/FO/checkout/index.ts:374]:
```suggestion
   * @param page {Page} Browser tab
   * @returns Promise<string>
```

File: tests/UI/pages/FO/checkout/index.ts [hunk 4]
```diff
@@ -328,6 +340,36 @@ class Checkout extends FOBasePage {
...
+   */
+  async getCustomerIdentity(page: Page): Promise<string> {
+    return this.getTextContent(page, this.personalInformationCustomerIdentity);
+  }
+
+  async getLogoutMessage(page: Page): Promise<string> {
```
Comment (Progi1984) [inline tests/UI/pages/FO/checkout/index.ts:386]:
```suggestion
  /**
   * @param page {Page} Browser tab
   * @returns Promise<string>
   */
  async getLogoutMessage(page: Page): Promise<string> {
```

File: tests/UI/pages/FO/cart/index.ts [hunk 5]
```diff
@@ -131,6 +137,18 @@ class Cart extends FOBasePage {
     this.alertPromoCode = '#promo-code > div > div > span';
   }
 
+  /*
+ Methods
+  */
+
+  /**
+   * Get no items in your cart message
+   * @param page {Page} Browser tab
```
Comment (Progi1984) [inline tests/UI/pages/FO/cart/index.ts:146]:
```suggestion
   * @param page {Page} Browser tab
   * @returns Promise<string>
```

**Review by Progi1984 — COMMENTED** [review]
Some feedbacks

File: tests/UI/pages/FO/cart/index.ts (hunk 5 above)
Comment (nesrineabdmouleh) [reply tests/UI/pages/FO/cart/index.ts:146]: Done :+1: 
Thx Franck :)

**Review by Progi1984 — APPROVED** [review]

**Review by boubkerbribri — APPROVED** [review]

**Comment by boubkerbribri** [comment]
Thanks @nesrineabdmouleh

**Pull request merged**